src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -8060,7 +8064,23 @@ private UsingStatementSyntax ParseUsingStatement()
             var closeParen = this.EatToken(SyntaxKind.CloseParenToken);
             var statement = this.ParseEmbeddedStatement();
 
-            return _syntaxFactory.UsingStatement(@using, openParen, declaration, expression, closeParen, statement);
+            return _syntaxFactory.UsingStatement(@using, awaitToken, openParen, declaration, expression, closeParen, statement);
+        }
+
+        private SyntaxToken ParseAwaitTokenInUsingOrForEach()
```

Comment:
>ParseAwaitTokenInUsingOrForEach [](start = 28, length = 31)

Please name it for what it does rather than where it is used, e.g. `ParseOptionalAwaitKeywordForAsyncStreams`